department-of-veterans-affairs/va.gov-cms
# VACMS-2453 media list images component

## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/board?assignees=kaise-lafrai&filterLogic=any&repos=154174777

## Testing done


## Screenshots


## QA steps

As an editor:
Login

1. Navigate here: admin/structure/media/manage/image/fields
    - [x] Confirm new 'field_description' field has been added with the following specifications:
            - [ ] add field_description to Image media type | plain text field | Char limit 300 | optional
2. Navigate to Paragraph page:
    - [x] Confirm the following specifications exist:
            - [ ] "Section Header" field_section_header | optional
            - [ ] "Images" field_images | entity reference to Media type "Image" | cardinality - unlimited
3. [x] Spec tool tests updated

## Definition of Done
